packages/core/src/components/essentials/tooltip-axis.ts

```diff
@@ -9,8 +9,8 @@ export class AxisChartsTooltip extends Tooltip {
 			return e.detail.items
 		}
 
-		const data = e.detail.data
-		if (!data.length || !data[0]) {
+		const { data } = e.detail
+		if (!data || !data.length || !data[0]) {
```

Comment:
Fix for https://github.com/carbon-design-system/carbon-charts/issues/1613

Reply:
we should have this kind of thing in a separate PR, but nbd for now